src/System.Runtime/ref/Configurations.props

```diff
@@ -3,7 +3,8 @@
   <PropertyGroup>
     <BuildConfigurations>
       netcoreapp;
+      netcoreappaot;
```

Comment:
Done. This could eventually be problematic since the field in question tells Roslyn that the runtime supports a feature that it doesn't, but I don't think we need to block on it for now.

Reply:
Runtimes don't have the ability to limit compile-time features.  You have to support that feature good enough so that folks can "light-down" at runtime.

Reply:
The feature in question is default interface implementations. Until that's actually built, I suspect the best we can do is a decent build error. As I said, we don't need to block for now since:
1. The framework doesn't use them yet, so it shouldn't affect testing
2. Nobody else uses them yet either since we haven't shipped a CoreCLR with it.